src/pybind/mgr/dashboard/services/cluster.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+from enum import Enum
+
+from .. import mgr
+
+
+class ClusterModel:
+
+    class Status(Enum):
+        INSTALLED = 0
+        POST_INSTALLED = 1
+
+    status: Status
+
+    def __init__(self, status=Status.INSTALLED.name):
+        self.status = self.Status[status]
+
+    def dict(self):
+        return {'status': self.status.name}
+
+    def to_db(self):
+        mgr.set_store('cluster/status', self.status.name)
+
+    @classmethod
+    def from_db(cls):
+        return cls(status=mgr.get_store('cluster/status', cls.Status.INSTALLED.name))
```

Comment:
Looks good. I see you preferred to store the full string rather than the enum number. Both approaches have pros and cons. Storing the Enum number is less intuitive if you have to debug the KV store. On the other hand, if we decide to change the name from POST_INSTALLED to WHATEVER, the Enum number wouldn't change. I'm ok with either.